app/src/main/java/at/irfc/app/data/remote/api/VotingApi.kt

```diff
@@ -14,6 +15,7 @@ interface VotingApi {
     @GET("votings/{votingId}")
     suspend fun getVoting(@Path("votingId") id: Long): VotingDto?
 
-    @POST("votings")
+    @POST("votes")
+    @Headers("Content-Type: application/json")
```

Comment:
Isn't that default? Or could that be set as default?

Reply:
It should be default, but it doesn't work without.
Ktor changed some internals of the request body serialization and I guess ktorfit did not yet adapt to that.